src/main.rs

```diff
@@ -41,6 +42,16 @@ fn main() -> Result<()> {
 
     println!("Running srsglass with user nation {}", args.user_nation);
 
+    let outfile = match args.outfile { 
+        Some(filename) => filename, 
+        None => { 
```

Comment:
```suggestion
    let outfile = match args.outfile {
        Some(filename) => filename,
        None => {
```
I think there's some trailing spaces at the end of these lines which got picked up by `cargo fmt`.